src/main/java/lotto/model/LottoGame.java

```diff
@@ -1,19 +1,44 @@
 package lotto.model;
 
-import lotto.model.vo.PurchaseCount;
-
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
 public class LottoGame {
 
+    private static final String SPLIT_REGEX = ",";
+    private static final int ZERO = 0;
+
     private final List<LottoGroup> lottoGroups;
 
-    public LottoGame(final PurchaseCount purchaseCount, final LottoMachine lottoMachine) {
-        this.lottoGroups = Stream.generate(lottoMachine::generate)
-                .limit(purchaseCount.getCount())
-                .collect(Collectors.toUnmodifiableList());
+    public LottoGame(final PurchaseCounts purchaseCounts, final List<String> manualLottoNumbersInput, final LottoMachine lottoMachine) {
+        if (purchaseCounts.getManualPurchaseCount() == 0) {
```

Comment:
```suggestion
        if (purchaseCounts.getManualPurchaseCount() == ZERO) {
```

상수는 최대한 활용해보면 좋을 것 같네요~